packages/typescriptlang-org/src/components/layout/main.scss

```diff
@@ -138,6 +138,10 @@ html.font-open-dyslexic {
   a {
     color: var(--link-color);
   }
+  pre code a {
+    // Don't interfere with syntax highlighting in code blocks
+    color: inherit;
+  }
```

Comment:
Please let me know if there's a better place/way to make this exception. I couldn't tell if there were any particular SASS conventions I should be following.